javascript-conditionals/main.js

```diff
@@ -0,0 +1,239 @@
+console.log("startsWithJ('Java'):", startsWithJ('Java'));
+// true
+
+// the log method of the console object called with two arguments a string and 'startsWithJ()' function with 'Kotlin' as the single arguement
+console.log("startsWithJ('Kotlin'):", startsWithJ('Kotlin'));
+// false
+
+// the log method of the console object called with two arguments a string and 'startsWithJ()' function with 'C#' as the single arguement
+console.log("startsWithJ('C#'):", startsWithJ('C#'));
+// false
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true; // returning the boolean data type 'true'
+  } else {
+    // closing curly brace for if code block
+    return false; // returning the boolean data type 'false'
+  } // closing else curly brace
+}
+
+// a new object literal is being assigned to 'bart' variable
+const bart = {
+  name: 'Bartholomew JoJo Simpson', // a string is being assigned to the name property
+  age: 10, // a interger is being assigned to the age property
+}; // closing code block
+
+// the log method of the console object called with two arguments a string and 'isOldEnoughToDrink()' function with bart object as a single arguement
+console.log('isOldEnoughToDrink(bart):', isOldEnoughToDrink(bart));
+// false
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true; // returning the boolean data type 'true'
+  } else {
+    // closing curly brace for if code block
+    return false; // returning the boolean data type 'false'
+  } // closing else curly brace
+}
+
+// a new object literal is being assigned to 'homer' variable
+const homer = {
+  name: 'Homer Jay Simpson', // a string is being assigned to the name property
+  age: 39, // a interger is being assigned to the age property
+}; // closing code block
+
+// the log method of the console object called with two arguments a string and 'isOldEnoughToDrive()' function with 'homer' object as a single arguement
+console.log('isOldEnoughToDrive(homer):', isOldEnoughToDrive(homer));
+// true
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age >= 16 && person.age >= 21) {
+    return true; // returning the boolean data type 'true'
+  } else {
+    // closing curly brace for if code block
+    return false; // returning the boolean data type 'false'
+  } // closing else curly brace
+}
+
+// a new object literal is being assigned to 'lisa' variable
+const lisa = {
+  name: 'Lisa Marie Simpson', // a string is being assigned to the name property
+  age: 8, // a interger is being assigned to the age property
+}; // closing code block
+
+// the log method of the console object called with two arguments a string and 'isOldEnoughToDrinkAndDrive()' function with 'homer' object as a single arguement
+console.log(
+  'isOldEnoughToDrinkAndDrive(lisa):',
+  isOldEnoughToDrinkAndDrive(lisa)
+);
+// false
+
+function categorizeAcidity(pH) {
+  if (pH >= 0 && pH <= 14) {
+    if (pH >= 0 && pH < 7) {
```

Comment:
Works, but the nested `if` statements isn't necessary. You're already checking if it's greater than 0, or less than 14 in another statements inside of here, so you don't need the nested.